p2p.py

```diff
@@ -68,6 +68,7 @@ def main(show_past_investments):
     logger.info("**** Aggregate available data ****")
     logger.info("**********************************")
     df_investments = aggregate_investment_data(investment_files)
+    df_investments["Passive income per month"] = df_investments[marketplace.OUTSTANDING_PRINCIPAL] * (( df_investments[marketplace.INTEREST_RATE] / 12 ) / 100)
```

Comment:
Can we add a function to the calculator? It will be better to reuse it in the dashboard.